src/components/Divider.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr
+    style={{
+      borderTopColor: 'rgba(0, 0, 0, .12)',
```

Comment:
Color prop not doing anything :P try `color || 'rgba(0, 0, 0, 0.12)'`

Also might be able to fit whole style prop on one line.